recipes/kalasiris/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "kalasiris" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bdcb44b71dd7c3a0cf3deb42d18a7516098cb2a372710433413c0d1ac9449db
+
+build:
+  noarch: python
+  number: 0
+  skip: True # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
This is technically a valid configuration but I suspect it's not what you intend.

At the moment this can only be built with `python >=3.6` but it can be installed into an environment with any Python version. I presume you also want to limit the runtime python version as well?